src/cdk/explorer/nodes/constructNode.ts

```diff
@@ -66,7 +66,8 @@ export class ConstructNode extends AWSTreeNodeBase {
                 entities.push(
                     new ConstructNode(
                         this,
-                        treeInspector.getDisplayLabel(child),
+                        //treeInspector.getDisplayLabel(child),
+                        treeInspector.isStateMachine(child)?treeInspector.getDisplayLabel(child)+` Visualize!` :treeInspector.getDisplayLabel(child),
```

Comment:
We want most of this code to reflect code that we will keep once we are ready to merge into the public repo. This looks like some test code, so we should remove this.

Reply:
Ternary operators typically have spaces in between each part: 
```
treeInspector.isStateMachine(child) ? treeInspector.getDisplayLabel(child) + ` Visualize!` :  treeInspector.getDisplayLabel(child)
```